src/soundcloud.js

```diff
@@ -0,0 +1,31 @@
+import { URL } from 'url';
+
+export const shouldTransform = string =>
+  new URL(string).host === 'soundcloud.com';
+
+const getSoundcloudIFrameSrc = string =>
+  `https://w.soundcloud.com/player` +
+  `?url=${string}` +
+  `&color=ff5500` +
+  `&auto_play=false` +
+  `&hide_related=true` +
+  `&show_comments=true` +
+  `&show_user=true` +
+  `&show_reposts=false` +
+  `&show_teaser=false` +
+  `&visual=true`;
+
+export const getHTML = string => {
+  const iframeUrl = getSoundcloudIFrameSrc(string);
+
+  return (
+    `<iframe` +
```

Comment:
Don't like the `+` sign tbh.
We're using [template literals](https://developer.mozilla.org/nl/docs/Web/JavaScript/Reference/Template_literals), but we're still using the `+` sign.

This feels a bit grim to me actually.

But I like the cleanliness of the separate lines too. 😕

Reply:
Let's loose the `+` signs and just use 1 long template literal instead to be consistent with the CodeSandbox and YouTube transformers

https://github.com/MichaelDeBoey/gatsby-remark-embedder/blob/373c71b273dede704f88615d9b224f5bb930ceae/src/codesandbox.js#L7-L11

https://github.com/MichaelDeBoey/gatsby-remark-embedder/blob/373c71b273dede704f88615d9b224f5bb930ceae/src/youtube.js#L50-L54

Reply:
Cool, updated! 

Personally I find the 1-line template literals tough to parse, but I also understand that the template literal and `+` signs make the code a lot more verbose. Happy to keep things close to the code style of the rest of the project tho.